app/api/auth.py

```diff
@@ -269,3 +272,20 @@ def change_password():
         "name": user.fullname if user.fullname else None,
         "password-changed": True
     })
+
+
+def authorized_access_ticket(ticket_function):
+    @wraps(ticket_function)
+    def wrapper(*args, **kwargs):
+        if current_identity.is_anonymous:
```

Comment:
This is not checking if the ticket is bought by the person. Only if the person is logged in?

Reply:
Check for current user's id also checked now.

Reply:
I can't see where it is implemented?

Reply:
Forgot to push changes. Updating right now.

Reply:
Updated.